.github/workflows/python-package.yml

```diff
@@ -71,3 +75,85 @@ jobs:
             flags: unittests # optional
             fail_ci_if_error: true # optional (default = false)
             verbose: true # optional (default = false)
+
+
+  Integration:
+
+    needs:
+    - Unit
+    # XXX: DO NOT MERGE THIS LINE
+    if: ${{ always() }}
+
+    # XXX: We should set this to some reasonable maximum prior to merge. The
+    # default 6 hour timeout is far too long.
+    timeout-minutes: 30
+
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # XXX: RESTORE THIS MATRIX PRIOR TO MERGE
+        python-version: ['3.7', ] #'3.8', '3.9', 'pypy3']
+        # XXX: Does each matrix variation get their own copy of the service
+        # containers? Test cross-talk may be an issue...
+        toxenv: ['redis', ] #'rabbitmq']
```

Comment:
This should probably actually be called `backend` since it's only the tail of the toxenv name.